### PR TITLE
Update redis host settings docs

### PR DESCRIPTION
**Pull request by urso**
- Doc update

## What does this PR do?

Output reference config files and redis output docs.

## Why is it important?

The redis host setting accepts passwords and URL schemas by now.
This change update the reference config files and docs.

## Checklist

~~- [ ] My code follows the style guidelines of this project~~
~~- [ ] I have commented my code, particularly in hard-to-understand areas~~
- [x] I have made corresponding changes to the documentation
- [x] I have made corresponding change to the default configuration files
~~- [ ] I have added tests that prove my fix is effective or that my feature works~~

## Related issues

- Relates #16206

### CODE REVIEW

File: auditbeat/auditbeat.reference.yml [hunk 1]
```diff
@@ -800,6 +800,8 @@ output.elasticsearch:
   # The list of Redis servers to connect to. If load-balancing is enabled, the
   # events are distributed to the servers in the list. If one server becomes
   # unreachable, the events are distributed to the reachable servers only.
+  # The hosts settings supports redis and rediss urls with custom password like
```
Comment (dedemorton) [inline auditbeat/auditbeat.reference.yml]: Change to "The hosts setting supports..."

File: libbeat/outputs/redis/docs/redis.asciidoc [hunk 2]
```diff
@@ -48,6 +48,13 @@ distributed to the servers in the list. If one server becomes unreachable, the e
 distributed to the reachable servers only. You can define each Redis server by specifying
 `HOST` or `HOST:PORT`. For example: `"192.15.3.2"` or `"test.redis.io:12345"`. If you
 don't specify a port number, the value configured by `port` is used.
+Each Redis server can be defined as a `IP:PORT` pair or as a `URL`. For
+example: `redis://localhost:6379`,or `rediss://localhost:6379`.
```
Comment (dedemorton) [inline libbeat/outputs/redis/docs/redis.asciidoc]: remove the comma before "or" (not needed here).

File: libbeat/outputs/redis/docs/redis.asciidoc [hunk 3]
```diff
@@ -48,6 +48,13 @@ distributed to the servers in the list. If one server becomes unreachable, the e
 distributed to the reachable servers only. You can define each Redis server by specifying
 `HOST` or `HOST:PORT`. For example: `"192.15.3.2"` or `"test.redis.io:12345"`. If you
 don't specify a port number, the value configured by `port` is used.
+Each Redis server can be defined as a `IP:PORT` pair or as a `URL`. For
```
Comment (dedemorton) [inline libbeat/outputs/redis/docs/redis.asciidoc]: I'd change this to say: 

```
Define each Redis server as an `IP:PORT` pair or a `URL`.
```

Note that pronunciation determines whether we use "a" or "an".

File: libbeat/outputs/redis/docs/redis.asciidoc [hunk 4]
```diff
@@ -48,6 +48,13 @@ distributed to the servers in the list. If one server becomes unreachable, the e
...
+example: `redis://localhost:6379`,or `rediss://localhost:6379`.
+Using URLs one can add a server specific password like as well. For example:
```
Comment (dedemorton) [inline libbeat/outputs/redis/docs/redis.asciidoc]: How about just: 

```
URLs can include a server-specific password. For example:
```

File: libbeat/outputs/redis/docs/redis.asciidoc [hunk 5]
```diff
@@ -48,6 +48,13 @@ distributed to the servers in the list. If one server becomes unreachable, the e
...
+`redis://:password@localhost:6379`.
+The `redis` scheme will disable the `ssl` settings for the host, while `rediss`
+will enforce TLS. In the later case the output will use the system certificate store if no `ssl`
```
Comment (dedemorton) [inline libbeat/outputs/redis/docs/redis.asciidoc]: To save a couple words and be more precise, say:

```
If `rediss` is specified and no `ssl` settings are configured, the output uses the system certificate store.
```

File: libbeat/outputs/redis/docs/redis.asciidoc (hunk 3 above)
Comment (urso) [reply libbeat/outputs/redis/docs/redis.asciidoc]: Changing to: `Configure each Redis server with an `IP:PORT` pair or with a `URL`.`
 
I get the `an 'IP:PORT'`, but why it says `a 'URL'` is beyond me :P

Comment (dedemorton) [reply libbeat/outputs/redis/docs/redis.asciidoc]: The English language likes to confuse people. :-) In this case, the pronunciation "you-are-ell" begins with a consonant sound, so "a" is the correct article to use. Crazy huh?

Comment (urso) [reply libbeat/outputs/redis/docs/redis.asciidoc]: TIL, thanks :)

**Pull request merged**